### PR TITLE
622 quick start data and sparkjobs

### PR DESCRIPTION
**Pull request by Zejnilovic**
Next in the series of quick start guides. Spark Jobs and Data quality.

### CODE REVIEW

File: _docs/2.0.0/usage/data-quick-start.md [hunk 1]
```diff
@@ -6,7 +6,35 @@ categories:
...
+
+but as Spline is under heavy development, we will postpone extensive documentation about it. Enceladus currently runs with version 0.3.X and it works fine out of the box. Spline 0.3.X just needs to be deployed next to Menas.
+
+Data for Spline are recorded even if the Spline UI is not up and running. This means they can be viewed later without the need to care bout it now.
+
+More about spline at [Spline](https://github.com/AbsaOSS/spline) Github.
+
+## _INFO files
+
+`_INFO` files our our way of tracking where the data came from and how much is there. It checks mainly that we did not loose any data on the way to standardization and conformance. All this made possibleby [Atum](https://github.com/AbsaOSS/atum). `_INFO` files should be placed withing the source directory together with the raw data.
```
Comment (benedeki) [inline _docs/2.0.0/usage/data-quick-start.md:34]:
```suggestion
`_INFO` files is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` files needs ti be placed within the source directory together with the raw data.
```

File: _docs/2.0.0/usage/data-quick-start.md [hunk 2]
```diff
@@ -6,7 +6,35 @@ categories:
...
+## Data Quality
+
+Data quality is all about 2 things:
+
+- Spline
+- _INFO files
+
+## Spline
+
+but as Spline is under heavy development, we will postpone extensive documentation about it. Enceladus currently runs with version 0.3.X and it works fine out of the box. Spline 0.3.X just needs to be deployed next to Menas.
```
Comment (benedeki) [inline _docs/2.0.0/usage/data-quick-start.md:26]: **Question:** I don't think that Spline 0.3.x has to be "deployed". Perhaps only MongoDb structures needs to be created?

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 3]
```diff
@@ -7,8 +7,97 @@ categories:
...
+- `dataset_versin` is a version of the dataset that has the correct schema and all the conformance rules you want
+- `report_date` represent the date on which the data landed in the HDFS (in raw) and also the next part of the path of to the data. So if in Menas you specified `/path/on/hdfs/raw` as your raw and input `2020-12-24` as a report date then the path where standardization will look for your files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing report versions.
+- `report_version` is the final part of the path on HDSF. With `report_date` we finished with `/path/on/hdfs/raw/2020/12/24/v<report_version>`. This is the location where standardization will look for raw data. 
+- `raw-format` and its specifis. Raw format tells the standardization in which format the data is on the HDFS and what are the specifics. CSV might have header, XML has row tag, etc. Here in the example we use row-tag. For more options for different types and run parameters see our [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run) or just run `--help`
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply _docs/2.0.0/usage/spark-jobs-quick-start.md:48]: header, row-tag, charset, etc.

File: _docs/2.0.0/usage/data-quick-start.md (hunk 2 above)
Comment (Zejnilovic) [reply _docs/2.0.0/usage/data-quick-start.md:26]: Spline creates the mongo structure as it goes. And it does not have to be deployed to run as per the next paragraph.

File: _docs/2.0.0/usage/spark-jobs-quick-start.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (benedeki) [reply _docs/2.0.0/usage/spark-jobs-quick-start.md:48]: I mean, you meant _specifics_? Type there 😉

File: _docs/2.0.0/usage/data-quick-start.md (hunk 2 above)
Comment (benedeki) [reply _docs/2.0.0/usage/data-quick-start.md:26]: Actually yeah, The Spline war has to be deployed... And configured both in Menas and Enceladus
Maybe worth clarifying?

Comment (Zejnilovic) [reply _docs/2.0.0/usage/data-quick-start.md:26]: In a quick start? I would rather not. In Deployement process `docs/2.0.0/deployment` sure thing. Can add it in this PR if you don't mind. Then maybe reference it here.

Comment (benedeki) [reply _docs/2.0.0/usage/data-quick-start.md:26]: Agreed - with the `docs/2.0.0/deployment` enhancement and referencing here.

File: _docs/2.0.0/usage/data-quick-start.md [hunk 4]
```diff
@@ -6,7 +6,35 @@ categories:
...
+
+Data for Spline are recorded even if the Spline UI is not up and running. This means they can be viewed later without the need to care about it now.
+
+More about spline at [Spline](https://github.com/AbsaOSS/spline) Github.
+
+## _INFO files
+
+`_INFO` file is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` files need to  be placed within the source directory together with the raw data.
```
Comment (benedeki) [inline _docs/2.0.0/usage/data-quick-start.md:34]:
```suggestion
`_INFO` file is our way of tracking where the data came from and how much data is there. It checks mainly that no data were lost on the way to and through standardization and conformance. All this is made possible by [Atum](https://github.com/AbsaOSS/atum). `_INFO` file needs to  be placed within the source directory together with the raw data.
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 5]
```diff
@@ -7,8 +7,97 @@ categories:
...
+--report-version <report_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+where:
+
+- `dataset_name` is the name gaven the dataset per [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide Guide
+- `dataset_version` is a version of the dataset to use, which should have the correct schema and all the conformance rules wanted
+- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the next part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` as a report date then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
```
Comment (benedeki) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:46]:
```suggestion
- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the suffix part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 6]
```diff
@@ -7,8 +7,97 @@ categories:
...
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <report_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+where:
+
+- `dataset_name` is the name gaven the dataset per [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide Guide
```
Comment (benedeki) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:44]:
```suggestion
- `dataset_name` is the name given the dataset per [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide Guide
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 7]
```diff
@@ -7,8 +7,97 @@ categories:
...
+- `dataset_version` is a version of the dataset to use, which should have the correct schema and all the conformance rules wanted
```
Comment (dk1844) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:45]:
```suggestion
- `dataset_version` is a version of the dataset to use, which should have the correct schema and all the desired conformance rules 
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 8]
```diff
@@ -7,8 +7,97 @@ categories:
...
+- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the next part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` as a report date then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
+- `report_version` is the final part of the path on HDSF. With `report_date` we finished with `/path/on/hdfs/raw/2020/12/24/v<report_version>`. This is the location where standardization will look for raw data.
+- `raw-format` and its specifics. Raw format tells the standardization in which format the data is on the HDFS and what are the specifics. CSV might have header, XML has row tag, etc. Here in the example we use row-tag. For more options for different types and run parameters see our [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run) or just run `--help`
```
Comment (dk1844) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:48]:
```suggestion
- `raw-format` and its specifics. Raw format tells the standardization which format the data is in on the HDFS and what are its specifics. CSV might have a header, XML has a row-tag, etc. Here in the example, we use the `row-tag`. For more options for different types and run parameters see our [run documentation](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run) or just run `--help`
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 9]
```diff
@@ -7,8 +7,97 @@ categories:
...
+--class za.co.absa.enceladus.conformance.DynamicConformanceJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+Here nothing new is added for the quick run. Of course there might be special options which are all documented in the [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```
Comment (dk1844) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:70]:
```suggestion
Here, nothing new is added for the quick run. Of course, there might be special options which are all documented in the [run documentation](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```

I think we should only use _documentation_ in the document, the plular form of the word, albeit technically possible, is rarely used. (also elsewhere in the doc).

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 10]
```diff
@@ -7,8 +7,97 @@ categories:
     - 'usage'
 redirect_from: /docs/usage/spark-jobs-quick-start
 ---
+{% capture docs_path %}{{ site.baseurl }}/docs/{{ page.version }}{% endcapture %}
+
+## Prerequsites
 
 This quick start guide presumes that you have went through :
```
Comment (dk1844) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:14]:
```suggestion
This quick start guide presumes that you have gone through :
```

File: _docs/2.0.0/usage/menas-quick-start.md [hunk 11]
```diff
@@ -76,3 +76,5 @@ _Don't mind the orange color. It only means the paths don't exist at the moment
 {% include image.html name="usage/menas_new_dataset.png" alt="Menas New Dataset" %}
 
 After clicking on _Save_, we have both the Dataset and accompanying Schema ready and we are able to run Standardization and Conformance.
+
+[**Next Data Qiality Quick Sart**]({{ site.baseurl }}/docs/{{ page.version }}/usage/data-quick-start)
```
Comment (dk1844) [inline _docs/2.0.0/usage/menas-quick-start.md:80]:
```suggestion
[**Next Data Quality Quick Sart**]({{ site.baseurl }}/docs/{{ page.version }}/usage/data-quick-start)
```

File: _docs/2.0.0/usage/spark-jobs-quick-start.md [hunk 12]
```diff
@@ -7,8 +7,97 @@ categories:
...
+<path to scripts>/run_conformance.sh \
+--deploy-mode <client/cluster> \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+For more options and arguments check the [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```
Comment (dk1844) [inline _docs/2.0.0/usage/spark-jobs-quick-start.md:103]: You left "documentations" here, but it is just a detail.

Comment (Zejnilovic) [reply _docs/2.0.0/usage/spark-jobs-quick-start.md:103]: Not just a detail. I did not see the comment under the suggestions. Fixed. Thank you

**Pull request merged**